extensions/2.0/Khronos/KHR_displaymapping_pq/README.md

```diff
@@ -0,0 +1,439 @@
+
+Apart from being widely supported and used in the TV / movie industry the perceptual quantizer is also embraced by the gaming community, with support in the engines from some of the major game companies.  
+For instance, game engines Frostbite and Lumberyard and also in specific games such as Destiny 2 and Call Of Duty.  
+
+## Internal range of light contribution values
+
+This section describes how light contribution values shall be handled.  
+
+When the KHR_displaymapping_pq extension is used all lighting and pixel calculations shall be done using the value 10000 (cd / m2) as the maximum ouput brightness.  
+This does not have an impact on color texture sources since they define values as contribution factor.  
+The value 10000 cd / m2 for an output pixel with full brightness is chosen to be compatible with the Perceptual Quantizer (PQ) used in the SMPTE ST 2084 transfer function.  
+
+When using this extension light contribution values shall be aligned to account for 10000 cd/m2 as max luminance, meaning that the dynamic range is 0 to 10000.    
+This means that content creators shall be aware of 10000 cd/m2 as the maximum brightness value range.  
+It does not mean that the display will be capable of outputing at this light luminance.  
+
+## Exporter considerations for light contribution values
+
+This section describes how an exporter shal handle lightsources, such as point, directional or environment lights, that are saved with the model.  
+
+A content creation tool supporting this extension shall sum upp light contribution for a scene before exporting to glTF, this can be a naive addition of all lights included in the scene that adds max values together.  
+If scene max light contribution is above 10000 cd / m2 there is a choice to either downscale light values before export or to set `sceneAperture` to values that will scale light contribution for a scene  
+
+Light contribution values above 10000 cd / m2 is discouraged and will be clamped by implementations if not adjusted by scene aperture.  
+
+## Display type
+
+This section describes how to decide if display type is HDR or SDR.  
+
+### HDR capable display
+
+
+If the framebuffer format and colorspace is known to the implementation then a format and colorspace shall be chosen to preserve the range and precision of the SMPTE ST 2084 transfer function.  
+If available, a framebuffer colorspace that is compatible with the color primaries of ITU BT.2020 shall be used.  
+If this colorspace is used the source color images must be converted from linear ITU BT.709 values to ITU BT.2020 linear values, where applicable depending on texture target and source format.  
+
+For HDR output,  a range extension value of 59.5208 and gamma of 2.4 shall be used.  
+
+**Implementation notes**
+
+Conversion of color primaries to ITU BT.2020 could be done after loading of a PNG/JPEG and after the image has been gamma expanded from sRGB to linear.  
```

Comment:
This note should be omitted as the initial image compression and EOTF decoding is already defined in the core spec.

Reply:
The current wording is:
```
Conversion of color primaries to ITU BT.2020 could be done after loading of a PNG/JPEG and after the image has been gamma expanded from sRGB to linear.
Another solution would be to perform render calculation in BT.709 colorspace and convert to target colorspace before writing out values to the framebuffer.
```
Is this ok with you?